### PR TITLE
Create open_port.sh

### PR DESCRIPTION
**Pull request by Loginbug**
Script to help to open a port for a client

### CODE REVIEW

**Comment by Loginbug** [comment]
Hei, any news for this?
I can also write docs to explain this!

**Comment by hwdsl2** [comment]
@Loginbug Thank you for contributing! Your script looks nice, I understand that it's intended for opening a port for VPN client(s) while they are connected to the VPN server. However, there are a few issues:
1. The duration that a particular VPN client stays connected to the VPN server could be short. If people use this script to open a port for a particular client, and later this client disconnects and another client connects and takes the same internal VPN IP, the open port would be incorrectly opened for that new VPN client.
2. Since there's no easy way to bind static IPs to each VPN client, (1) above would be a security concern for users. Even if we add a feature to the script to close the opened port, the particular VPN client may no longer be in the list of connected clients when the script is run again. An alternative is to check/parse the current list of open ports from IPTables output, but that may be difficult.
3. Advanced users may have customized the IPsec/L2TP and/or IPsec/XAuth ("Cisco IPsec") VPN subnets, so they are no longer `192.168.42.0/24` and `192.168.43.0/24`. The script cannot easily handle this case.

In summary, I think this (how to open ports for VPN clients) may be better suited for an entry in the troubleshooting section of README [1], rather than an automated script.

[1] https://github.com/hwdsl2/setup-ipsec-vpn/blob/master/docs/clients.md#troubleshooting
